tfe/organization_members_helpers.go

```diff
@@ -0,0 +1,41 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+func fetchOrganizationMembers(client *tfe.Client, orgName string, options tfe.OrganizationMembershipListOptions) ([]map[string]string, []map[string]string, error) {
```

Comment:
I'm feeling like this function shouldn't accept an arbitrary ListOptions parameter. Reason being, you always want to start at page 1 and return all pages, plus the shape of the return value determines which include params you need in the first place.

Reply:
I don't have a strong opinion about this one so I'll update it